CaloCluster/fcl/prolog.fcl

```diff
@@ -13,6 +13,12 @@ END_PROLOG
 
 BEGIN_PROLOG
 
+CaloClusterCompare : {
```

Comment:
Probably the best location for this analyzer is in the Analysis directory. CaloCluster should only keep modules and algorithms related to the reconstruction.